### PR TITLE
fix: reorganizing enterprise customer django admin

### PR DESCRIPTION
**Pull request by yuanyuan-git-tech**
https://2u-internal.atlassian.net/browse/ENT-7322
**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.
- [ ] [Version](https://github.com/openedx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [ ] [Changelog](https://github.com/openedx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/openedx/edx-enterprise/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [GitHub Actions](https://github.com/openedx/edx-enterprise/actions), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [edx-platform](https://github.com/openedx/edx-platform) to upgrade dependencies (including edx-enterprise)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.

### CODE REVIEW

File: enterprise/migrations/0175_auto_20230627_2027.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='auth_org_id',
+            field=models.CharField(blank=True, help_text="Enterprise customer's authentication ID if leveraging a third party platform such as Auth0 for authentication.", max_length=80, null=True, verbose_name='Third Party Auth Org ID:'),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='contact_email',
+            field=models.EmailField(blank=True, help_text='Email address presented on learner portal as public point of contact from customer organization.', max_length=254, null=True, verbose_name='Customer admin contact email:'),
```
Comment (kiram15) [inline enterprise/migrations/0175_auto_20230627_2027.py:29]: Ask Mike about the colons? I think it's weird that he has them on some fields and not others in the Figma and want to make they are intentional

File: enterprise/models.py [hunk 2]
```diff
@@ -223,41 +244,44 @@ class Meta:
...
+            "or through an external process."
         )
     )
 
     enable_audit_enrollment = models.BooleanField(
+        "Enable audit enrollment for learning platform learners",
         default=False,
         help_text=_(
-            "Specifies whether the audit track enrollment option will be displayed in the course enrollment view."
+            "Allows learners enrolling through learning latforms to select the audit track."
```
Comment (kiram15) [inline enterprise/models.py:271]: latforms -> platforms

File: enterprise/models.py [hunk 3]
```diff
@@ -370,9 +391,10 @@ class Meta:
     )
 
     contact_email = models.EmailField(
+        "Customer admin contact email:",
         null=True,
         blank=True,
-        help_text=_("Email to be displayed as public point of contact for enterprise.")
+        help_text = _("Email address presented on learner portal as public point of contact from customer organization.")
```
Comment (kiram15) [inline enterprise/models.py:397]: Can you remove the extra spaces around the equals for this instance and the others? Just to remain consistent

**Review by kiram15 — COMMENTED** [review]
Just a couple nits!

File: enterprise/migrations/0175_auto_20230627_2027.py (hunk 1 above)
Comment (yuanyuan-git-tech) [reply enterprise/migrations/0175_auto_20230627_2027.py:29]: Settings that had a dropdown or free text field have a colon, where checkboxes did not. However, Mike told me he missed a couple of the colons in the wireframe for the options with dropdowns/text fields. He just added two colons back to a couple of those options in the wireframe.

**Comment by yuanyuan-git-tech** [comment]
Screenshots from the new page
Screenshot 2023-06-28 at 9 36 25 AM
Screenshot 2023-06-28 at 9 36 46 AM
Screenshot 2023-06-28 at 9 39 23 AM
Screenshot 2023-06-28 at 9 39 34 AM

**Pull request closed without merging**